libs/s25main/desktops/dskLobby.cpp

```diff
@@ -73,7 +74,8 @@ dskLobby::dskLobby()
     // Chatfenster-Edit
     AddEdit(21, DrawPoint(20, 530), Extent(500, 22), TC_GREY, NormalFont);
 
-    AddTimer(30, 5000);
+    using namespace std::chrono_literals;
+    AddTimer(30, 5s);
```

Comment:
we should change the id to a define here too?

Reply:
Yes we should do that for all desktops. Once someone got some time, not now. To much work...